src/sourmash/sourmash_args.py

```diff
@@ -393,14 +393,27 @@ def load_pathlist_from_file(filename):
     try:
         with open(filename, 'rt') as fp:
             file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+        if len(file_list) == 0:
+            raise ValueError("pathlist is empty")
+        badFormat = False
+        duplicate = set()
+        for checkfile in file_list:
+            if not os.path.exists(checkfile):
+                badFormat = True
+            if checkfile not in duplicate:
```

Comment:
This `if` is unnecessary! You can just say `duplicate.add(checkfile)` - it's actually faster, in addition to being more concise!

Reply:
Which `if` is this? is it the one on line 401, or the one on line 403?

Reply:
the only `if ` that involves `duplicate` is on line 403.